specification/appconfiguration/resource-manager/readme.md

```diff
@@ -0,0 +1,89 @@
+# AppConfiguration
+
+> see https://aka.ms/autorest
+
+This is the AutoRest configuration file for AppConfiguration.
+
+---
+
+## Getting Started
+
+To build the SDK for AppConfiguration, simply [Install AutoRest](https://aka.ms/autorest/install) and in this folder, run:
+
+> `autorest`
+
+To see additional help and options, run:
+
+> `autorest --help`
+
+---
+
+## Configuration
+
+### Basic Information
+
+These are the global settings for the AppConfiguration API.
+
+``` yaml
+openapi-type: arm
+tag: package-2019-02-01-preview
+```
+
+### Tag: package-2019-02-01-preview
+
+These settings apply only when `--tag=package-2019-02-01-preview` is specified on the command line.
+
+``` yaml $(tag) == 'package-2019-02-01-preview'
+input-file:
+- Microsoft.AppConfiguration/preview/2019-02-01-preview/appconfiguration.json
+```
+
+---
+
+# Code Generation
+
+## Swagger to SDK
+
+This section describes what SDK should be generated by the automatic system.
+This is not used by Autorest itself.
+
+``` yaml $(swagger-to-sdk)
+swagger-to-sdk:
+  - repo: azure-sdk-for-python
+  - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-net
+  - repo: azure-sdk-for-js
+  - repo: azure-sdk-for-node
```

Comment:
could you update this like to be azure-sdk-for-ruby instead of node?
I see that ruby is missing, and the lines below are after_scripts for ruby. Node is getting deprecated, so no need to include it.